roles/dg-config/tasks/main.yml

```diff
@@ -0,0 +1,198 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: DG | SQL to add standby logs
+  set_fact:
+    sql_for_dg:  |
+      alter system set dg_broker_start = true;
+      alter system set standby_file_management = auto;
```

Comment:
What if the primary is RAC?  Should standby_file_management be set on the other node too?

Reply:
I will change to use ...sid='*'